test/reporters/xunit.spec.js

```diff
@@ -194,15 +194,13 @@ describe('XUnit reporter', function() {
       it('should call write with line', function() {
         stdoutWrite = process;
         process = false; // eslint-disable-line no-native-reassign, no-global-assign
-        var cachedConsoleLog = console.log;
-        console.log = function(string) {
-          stdout.push(string);
-        };
 
+        var savedConsoleLog = reporters.Base.consoleLog;
```

Comment:
as above, stubbing & un-stubbing should happen in hooks

Reply:
Similar issue to above, if `reporters.Base.consoleLog` is not back by end of test it breaks the reporter running the test. Any ideas?

Reply:
That isnt going to be possible. Please see above for reason, except this cant `callThrough()`.

Reply:
See below